README.md

```diff
@@ -2,35 +2,43 @@
  # Usage on Local Environment
  
  ## Pre-Requisites
- ###    Create Snapshot Database
- ###    Create Snapshot Users 
- ###    Update Scripts with Snapshot Database Credentials
+ ### Create Snapshot Database & Users
+Ensure a local Postgres database with access granted to user `ccd` is available on local machine
+      
+ ### Configure the database properties to be used
```

Comment:
i think this section goes above Local setup as it's common to both local and other envs

Reply:
Fixed in c8d414cec56d532e036df0aea7191c12e884c31f